onnx/checker.cc

```diff
@@ -595,19 +595,25 @@ void check_attribute(const AttributeProto& attr, const CheckerContext& ctx, cons
   }
 }
 
+void print_warning_if_has_experimental(const std::unordered_set<std::string>& existed_experimental_ops) {
```

Comment:
Minor nit: `used_experimental_ops` would be a slightly better name than `existed_experimental_ops`.

Reply:
Updated.